MOBILE_CLIENT/ANDROID/_LOW/APK_DEBUG_SYMBOL_PRESENT/description.md

```diff
@@ -0,0 +1,3 @@
+The application should provide as little explanatory information as possible with the compiled code. Metadata such as debugging information, line numbers, and descriptive function or method names make the binary or byte-code easier to reverse engineer.
```

Comment:
This is still not telling the use what is the issue.

We should simply start with the application ships with debug symbols.